PWGMM/Lumi/Tasks/LumiFDDFT0.cxx

```diff
@@ -0,0 +1,305 @@
+                  full::isFT0, full::TCMTriggerFT0,
+                  full::TimeAFT0, full::TimeCFT0,
+                  full::ChargeAFT0, full::ChargeCFT0);
+} // namespace o2::aod
+
+struct LumiFDDFT0 {
+  Produces<o2::aod::EventInfo> rowEventInfo;
+  Service<o2::ccdb::BasicCCDBManager> ccdb;
+  const char* ccdbpath_grp = "GLO/Config/GRPMagField";
+  const char* ccdburl = "http://alice-ccdb.cern.ch";
+  int mRunNumber;
+
+  Configurable<Long64_t> ftts{"ftts", 1668080173000, "First time of time stamp"};
+  Configurable<int> nContribMax{"nContribMax", 2500, "Maximum number of contributors"};
+  Configurable<int> nContribMin{"nContribMin", 10, "Minimum number of contributors"};
+
+  HistogramRegistry histos{
+    "histos",
+    {
+      {"vertexx", "", {HistType::kTH1F, {{1000, -1, 1, "x"}}}},                                         //
+      {"vertexy", "", {HistType::kTH1F, {{1000, -1, 1, "y"}}}},                                         //
+      {"timestamp", "", {HistType::kTH1F, {{20000, 0, 2e7, "t"}}}},                                     //
+      {"vertexx_timestamp", "", {HistType::kTH2F, {{10000, 130e9, 140e9, "t"}, {2000, -1, 1, "x"}}}},   //
+      {"vertexy_timestamp", "", {HistType::kTH2F, {{10000, 130e9, 140e9, "t"}, {2000, -1, 1, "y"}}}},   //
+      {"chisquare", "", {HistType::kTH1F, {{1000, 0, 100, "#chi^{2}"}}}},                               //
+      {"vertexx_Refitted", "", {HistType::kTH1F, {{1000, -1, 1, "x"}}}},                                //
+      {"vertexy_Refitted", "", {HistType::kTH1F, {{1000, -1, 1, "y"}}}},                                //
+      {"vertexx_Refitted_timestamp", "", {HistType::kTH2F, {{1000, 0, 4e6, "t"}, {2000, -1, 1, "x"}}}}, //
+      {"vertexy_Refitted_timestamp", "", {HistType::kTH2F, {{1000, 0, 4e6, "t"}, {2000, -1, 1, "y"}}}}, //
+      {"chisquare_Refitted", "", {HistType::kTH1F, {{1000, 0, 100, "#chi^{2}"}}}},                      //
+      {"vertexx_Refitted_vertexx", "", {HistType::kTH2F, {{1000, -1, 1, "x"}, {1000, -1, 1, "rx"}}}},   //
+      {"vertexy_Refitted_vertexy", "", {HistType::kTH2F, {{1000, -1, 1, "y"}, {1000, -1, 1, "ry"}}}}    //
+    }};
+
+  bool doPVrefit = true;
+
+  void init(InitContext&)
+  {
+    ccdb->setURL(ccdburl);
+    ccdb->setCaching(true);
+    ccdb->setLocalObjectValidityChecking();
+    uint64_t now = std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count();
+    ccdb->setCreatedNotAfter(now);
+    mRunNumber = 0;
+  }
+
+  void process(soa::Join<aod::Collisions, aod::EvSels>::iterator const& collision, aod::FDDs const& fdds, aod::FT0s const& ft0s, aod::BCsWithTimestamps const&,
+               o2::soa::Join<o2::aod::Tracks, o2::aod::TrackSelection,
+                             o2::aod::TracksCov, o2::aod::TracksExtra,
+                             o2::aod::TracksDCA> const& tracks,
+               o2::soa::Join<o2::aod::Tracks, o2::aod::TracksCov,
```

Comment:
There is no reason to subscribe to Tracks twice, especially since no filtering is actually applied and only `unfiltered_tracks` is used in the process function.